.gitattributes

```diff
@@ -1,3 +1,4 @@
 *.generated.swift linguist-generated=true
 */Tests/**/Resources/ReferenceImages/**/*.png filter=lfs diff=lfs merge=lfs -text
 wire-ios/Wire-iOS[[:space:]]Tests/ReferenceImages/**/*.png filter=lfs diff=lfs merge=lfs -text
+WireDomain/Frameworks/WireBackup.xcframework.zip filter=lfs diff=lfs merge=lfs -text
```

Comment:
Temporarily checking in the binary into the repo (LFS) until it is deployed somewhere.